From dc3bbe227d7af1dbb87caa4aca932b4710cbd3e0 Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Tue, 13 Aug 2019 20:31:28 +0200
Subject: [PATCH 19/23] gtkfilechooserwidget: Expand the filename entry

Drop the minimum number of characters for the filename entry, and
instead expand it. This helps it fit in narrow screens.
---
 gtk/gtkfilechooserwidget.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkfilechooserwidget.c b/gtk/gtkfilechooserwidget.c
index 4f136068e0..bed8736640 100644
--- a/gtk/gtkfilechooserwidget.c
+++ b/gtk/gtkfilechooserwidget.c
@@ -2625,7 +2625,7 @@ location_entry_setup (GtkFileChooserWidget *impl)
   _gtk_file_chooser_entry_set_action (GTK_FILE_CHOOSER_ENTRY (priv->location_entry), priv->action);
   _gtk_file_chooser_entry_set_file_filter (GTK_FILE_CHOOSER_ENTRY (priv->location_entry),
                                            priv->current_filter);
-  gtk_entry_set_width_chars (GTK_ENTRY (priv->location_entry), 45);
+  gtk_widget_set_hexpand (priv->location_entry, TRUE);
   gtk_entry_set_activates_default (GTK_ENTRY (priv->location_entry), TRUE);
 }
 
-- 
2.28.0

